src/pivot-table/components/cells/styles.tsx

```diff
@@ -67,3 +67,34 @@ export const StyledLabel = styled("div", {
   fontStyle,
   textDecoration,
 }));
+
+export const AdjusterHitArea = styled("div", {
+  shouldForwardProp: (prop: string) => prop !== "isLastColumn",
+})(({ isLastColumn = false }: { isLastColumn: boolean }) => ({
+  display: "flex",
+  position: "absolute",
+  height: "100%",
+  top: 0,
+  left: `100%`,
```

Comment:
```suggestion
  left: "100%",
```

Reply:
it shouldn't be ther at all, we set left by inline styling, since it determines the positon